app/src/main/java/com/kickstarter/extensions/ToastExt.kt

```diff
@@ -0,0 +1,23 @@
+package com.kickstarter.extensions
+
+import android.content.Context
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import com.kickstarter.R
+
+fun showErrorToast(applicationContext: Context, root: ViewGroup, message: String) {
+    val layout: ViewGroup = LayoutInflater.from(applicationContext).inflate(R.layout.toast_error, root, false) as ViewGroup
+    val text: TextView = layout.findViewById(R.id.toast_text_view)
```

Comment:
Maybe `textView` would be a better name,  instead of simply `text`.

Reply:
I stole this code lol https://developer.android.com/guide/topics/ui/notifiers/toasts#CustomToastView